cmd/deployment-check/README.md

```diff
@@ -110,20 +110,26 @@ spec:
   podSpec:
     containers:
       - name: deployment
-        image: kuberhealthy/deployment-check:v1.6.2
-        imagePullPolicy: IfNotPresent
+        image: kuberhealthy/deployment-check:v1.8.0
+        imagePullPolicy: Always
         env:
           - name: CHECK_DEPLOYMENT_REPLICAS
             value: "6"
           - name: CHECK_DEPLOYMENT_ROLLING_UPDATE
             value: "true"
+          - name: TOLERATION_VALUE
+            value: ""
+          - name: NODE_SELECTOR
+            value: ""
         resources:
           requests:
-            cpu: 15m
+            cpu: 25m
```

Comment:
any reason cpu was upped here?

Reply:
no. I believe some of these changes we are seeing came from pulling and merging master to keep my code up to date. I will move it back to 15m